src/scss/_typography.scss

```diff
@@ -37,15 +37,14 @@ body {
 }
 
 @font-face {
-  font-family: DistinctDisplay;
+  font-family: DistinctDisplayItalic;
```

Comment:
Why are we changing these back? 

By doing this you would have to change the font family of any item you would want to add italics or bold? Since in Phillips-public and eventually in Remix we will have a the typography file declare all section headers to have this font we can just add styling like bold and italic through traditional means.